webapp/src/selectors/index.tsx

```diff
@@ -27,3 +27,11 @@ export const getRhsState = (state: any): {isSidebarOpen: boolean} => {
 export const getSubscribeModalState = (state: any): SubscribeModalState => {
     return state[pluginPrefix].openSubscribeModalSlice;
 };
+
+export const getApiRequestCompletionState = (state: any): ApiRequestCompletionState => {
```

Comment:
Fix the type of state.

Reply:
this has been fixed in later PR will update the type here when synced or merged